lib/main.js

```diff
@@ -8,6 +8,8 @@ const {Cu} = require("chrome");
 Cu.import("resource://gre/modules/ClientID.jsm");
 Cu.import("resource://gre/modules/Task.jsm");
 
+const PING_ENDPOINT = "https://onyx_tiles.stage.mozaws.net/v3/links/activity-stream";
```

Comment:
Should this be in a env-specific config or something? It seems weird that we'd always send data to a staging server from a prod environment.